### PR TITLE
Add complex support for tan

### PR DESCRIPTION
**Pull request by yongtang**
At the moment tan in tensorflow supports float and double, but not complex types.

This fix adds the support for tan, and removes related TODO in the test

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

**Comment by martinwicke** [comment]
@yifeif I am concerned about copybara here -- can it get stuck? Or what is the failure mode here?

**Comment by qlzh727** [comment]
I think its waiting for at least one approval from reviewer.

**Review by martinwicke — APPROVED** [review]

**Pull request merged**
